packages/amplify-provider-awscloudformation/src/push-resources.ts

```diff
@@ -894,6 +894,29 @@ const handleCfnFiles = (eventMap: EventMap, category: string, resource: string,
   });
 };
 
+const getNestedStackParameters = async (context: $TSContext, categories: Array<string>, amplifyMeta: any) => {
+  const s3 = await S3.getInstance(context);
+  const currentEnvName = context.amplify.getEnvInfo().envName;
+  const urlParametersMap = {};
+  for (const category of categories) {
+    const resources = Object.keys(amplifyMeta[category]);
+    for (const resource of resources) {
+      const resourceDetails = amplifyMeta[category][resource];
+      if (resourceDetails.providerMetadata) {
+        const templateURL = resourceDetails.providerMetadata.s3TemplateURL;
```

Comment:
This only accounts for the "main" template within each category. However auth triggers, API Gateway policies and AppSync all create multiple nested stacks (and in the case of AppSync, more layers of nested stacks) that wouldn't be captured here.